### PR TITLE
Asset Fingerprinting for Kahuna

### PR DESCRIPTION
**Pull request by kenoir**
In order to cache-bust assets when serving through Cloudfront.

Depended on by https://github.com/guardian/grid-infra/pull/244

### CODE REVIEW

**Comment by kenoir** [comment]
Tried this in TEST succesfully.

File: kahuna/app/views/main.scala.html [hunk 1]
```diff
@@ -15,8 +15,8 @@
     <link rel="preconnect" href="@mediaApiUri"/>
     <link rel="preconnect" href="@authApiUri"/>
 
-    <link rel="shortcut icon" type="image/png" href="@routes.Assets.at("images/grid-favicon-32.png")"/>
-    <link rel="assets" href="@routes.Assets.at("")"/>
+    <link rel="shortcut icon" type="image/png" href="@routes.Assets.versioned("images/grid-favicon-32.png")"/>
```
Comment (akash1810) [inline kahuna/app/views/main.scala.html]: will we have multiple versions of the favicon 😄

**Comment by akash1810** [comment]
👍

File: kahuna/app/views/main.scala.html (hunk 1 above)
Comment (kenoir) [inline kahuna/app/views/main.scala.html]: I thiiiiink the way asset fingerprinting works is that we'll have multiple versions only when the favicon changes.

**Comment by ShaunYearStrong** [comment]
👍

**Pull request merged**